docs/Data Lake.md

```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
 
 ![Preliminary hitlist](../img/Discover/image13.png)
 
+##DepMap CCLE
+
+###Opening the app
+
+Upon opening the Discover portal on Polly, choose the Depmap data repository. The page should look something like this.
+
+![Selecting app](../img/DepMapCCLE/image1.png)
+
+After selecting the Depmap repository, you’ll be able to see a dashboard and the DepMap CCLE app icon. The dashboard will give you an overview of the repository, for instance, the information of different cell lines in the  repository, distribution of tissue lineages, metastatic status of samples, etc. among the datasets.
```

Comment:
Depmap repository in italics
....along with the DepMap CCLE *application (Italics)

Reply:
changed